scripts/combine/setupCombine.py

```diff
@@ -320,22 +333,55 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
         logger.info("Using option --doStatOnly: the card was created without nuisance parameters")
         return cardTool
 
-    if args.doStatOnly and constrainMass:
-        # no mass weight uncertainty for stat only fits if mass weight is a nuisance (e.g. unfolding, xsec, ...)
-        return cardTool
-
     cardTool.addSystematic(f"massWeight{label}",
                            processes=signal_samples_forMass,
                            group=f"massShift{label}",
                            noi=not constrainMass,
                            skipEntries=massSkip,
                            mirror=False,
-                           #TODO: Name this
                            noConstraint=not constrainMass,
                            systAxes=["massShift"],
                            passToFakes=passSystToFakes,
     )
 
+    if args.fitMassDiff:
+        suffix = "".join([a.capitalize() for a in args.fitMassDiff.split("-")])
+        mass_diff_args = dict(
+            name=f"massWeight{label}",
+            processes=signal_samples_forMass,
+            rename=f"massDiff{suffix}{label}",
+            group=f"massDiff{label}",
+            systNameReplace=[("Shift",f"Diff{suffix}")],
+            skipEntries=[(m[0].replace("50MeV", "100MeV"),) for m in massSkip],
```

Comment:
Probably nicer if the calculation of massSkip above becomes a function in syst_tools that takes 100 or 50 as an arg

Reply:
done